packages/core/src/routes/connector.ts

```diff
@@ -59,6 +67,57 @@ export default function connectorRoutes<T extends AuthedRouter>(router: T) {
     }
   );
 
+  router.post(
+    '/connectors/:connectorId',
+    koaGuard({
+      params: object({ connectorId: string() }),
+      body: Connectors.createGuard.omit({
+        id: true,
+        enabled: true,
+        connectorId: true,
+        createdAt: true,
+      }),
+    }),
+    async (ctx, next) => {
+      const {
+        params: { connectorId },
+        body: { config, metadata },
+        body,
+      } = ctx.guard;
+
+      const logtoConnectors = await loadConnectors();
+      const standardConnector = logtoConnectors.find(({ metadata: { id } }) => id === connectorId);
+
+      if (!standardConnector) {
+        throw new RequestError({
+          code: 'entity.not_found',
+          status: 404,
+        });
+      }
+
+      const { count } = await countConnectorByConnectorId(connectorId);
+      console.log(count, standardConnector.metadata.isStandard);
+      assertThat(
+        count === 0 || standardConnector.metadata.isStandard === true,
+        'connector.multiple_instances_not_supported'
+      );
+
+      assertThat(!config || !metadata, 'connector.config_and_metadata_empty');
+
+      if (config) {
```

Comment:
if `config` exist and pass the validation, this connector should be "enabled"?

Reply:
This logic aligns to PATCH /connectors/:id, which validates the `config` and still needs to manually enable the connector.